README.md

```diff
@@ -143,26 +141,23 @@ var gpio = require("gpio");
 var gpio22, gpio4, intervalTimer;
 
 // Flashing lights if LED connected to GPIO22
-gpio22 = gpio.open({
-    pin: 22,
-    ready: function() {
-        intervalTimer = setInterval(function() {
-            gpio22.set();
-            setTimeout(function() { gpio22.reset(); }, 500);
-        }, 1000);
-    }
+gpio22 = gpio.open({ pin: 22 }, function(err, self) {
```

Comment:
good, isnt direction missing here ?

Reply:
Default direction is _out_ (noted near the top of the doc). That said, I'll update to specify direction for clarity.